### ISSUE TITLE
Socket Errors.

### ISSUE THREAD

**Issue by NaPurHab** [BiglySoftware/BiglyBT#2926]
I am getting the following errors:

ERROR: socket Selector.open() for 'TCP network manager reader' failed 10 times in a row, aborting.
This App / Java is likely being firewalled!

ERROR: socket Selector.open() for 'TCP network manager writer' failed 10 times in a row, aborting.
This App / Java is likely being firewalled!

ERROR: socket Selector.open() for 'TCP Connect/Disconnect Manager' failed 10 times in a row, aborting.
This App / Java is likely being firewalled!

![image](https://user-images.githubusercontent.com/37715552/229369643-df7fba3d-1f99-444d-9f48-b01480e6c8a9.png)

As you can see, already upto 32 errors. They just keep repeating.

I did discover in a test, that going back to 3.3 release got rid of the errors.  When I turned Beta back on and updated, the errors came back.   So somewhere between 3.3 release and B25 this happened.

What caused this to show up is that my VPN disabled all OLD clients and forced upgrade to a NEW version.  New VPN version has lots of issues.    Here is what B25 is doing:

![image](https://user-images.githubusercontent.com/37715552/229369883-9d96c7ea-9a77-46f3-b999-fc18d1bd3e3f.png)

And here is Release 3.3, where the issue doesn't show.:

![image](https://user-images.githubusercontent.com/37715552/229370130-0da18710-32a3-4293-8f3f-6b621a17ed9b.png)

Oh another error that has been popping up:

![image](https://user-images.githubusercontent.com/37715552/229370068-9130005d-7205-4ed6-9c7d-f80dd475151f.png)

So lots happening, Ideas?

Java 1.8.0_361 (64 bit)
  Oracle Corporation
c:\program files\java\jre1.8.0_361

SWT v4942r22, win32, zoom=100, dpi=96
Windows 10 v10.0, amd64 (64 bit)
B3.3.0.1_B25/4 az3

**Comment by parg** [comment]
No idea, nothing's changed in BiglyBT so your problem lies elsewhere

**Comment by parg** [comment]
So you had to update your VPN, these issues started appearing and you blame BiglyBT. Surely the more obvious thing to blame is the VPN upgrade right?

**Comment by NaPurHab** [comment]
![image](https://user-images.githubusercontent.com/37715552/229372709-da1f142b-9126-4a1a-a67e-bc294ea02135.png)

So why is this happening.  This is BBT creating multiple copies.   And torrent Opens from browsers also cause multiple copies.

**Comment by parg** [comment]
I would imagine that your nice new VPN is blocking BiglyBT from listening on the port it uses to communicate with other instances of itself

**Comment by NaPurHab** [comment]
On another note, the following is also happening.  In Bind to local IP:

{pureVPN)||(WireGuard\sTunnel)     doesn't work.  But,
ppp1[0]||net11[0]     does work.

Damb odd. This is internal to BBT. So should not be affected.

**Comment by parg** [comment]
Got to go and do something other than stare at my computer, laters

**Comment by NaPurHab** [comment]
> I would imagine that your nice new VPN is blocking BiglyBT from listening on the port it uses to communicate with other instances of itself

I have a port forwarding option.  What external port do I need to enable?

**Comment by NaPurHab** [comment]
Ok, I got it working. Sort of?

I had to OPEN All ports on my VPN to port forwarding.  Then to get Prowlarr, Radarr and Sonarr to work. I had to put the VPN IP Address in the HOST settings in each program.  And in searching on the Web, I found that while BBT is running on Split Tunneling. The ARR programs must not be on the VPN. However, Localhost simply does not work at the moment.  Would be nice if I could find a way to grab the BIND TO address from BBT and update the HOST entries in the ARR programs. I tried putting an entry in the windows hosts for `localbind` and that did not work. Plus I would still have to update the HOSTS files every time.
